routes/routes.js

```diff
@@ -9,8 +9,17 @@ var storage = multer.diskStorage({
   },
   filename: function (req, file, cb) {
     console.log(file);
-    let newName = Date.now() + "_" + file.originalname;
-    cb(null, newName);
+    let newName = `${Date.now()}_${file.originalname}`;
```

Comment:
the variable name can me more efficient if you change it to `**newFileName**` @mukeshsin

Reply:
ok